src/style.css

```diff
@@ -1 +1,80 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+:root {
+  --main-color: #00acdc;
+}
+
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
Don't use * selector for zeroing out your margins or paddings. It's still inefficient for browser to read your web document